.github/workflows/php-test-plugins.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         coverage: [false]
         include:
           - php: '7.4'
-            wp: '6.5'
+            wp: '6.6'
```

Comment:
The `parentLayout` added in WP 6.6. See https://github.com/WordPress/WordPress/blob/e174cbef2e3b7f1901892cb501f683c6e122231b/wp-includes/block-supports/layout.php#L926-L943.

Once we bump the version in #1682 it will fix the issue.

Reply:
Let's open a separate PR for this change

Reply:
PR #1683 already opened but it currently blocked because of Felix's feedback. I added comment on that issue.

Reply:
This is a good reason to bump the minimum requirement to 6.6. The need for a feature only available in a certain WordPress version is a good rationale for bumping the requirement, as long as it's in line with our version policy - which it is, given the latest version is 6.7.

Reply:
https://github.com/WordPress/performance/pull/1683 is merged and the feature branch refreshed, so this should be good.